frontend/src/components/CalcSpectrum.tsx

```diff
@@ -201,7 +204,7 @@ export const CalcSpectrum: React.FC = () => {
         link.href = url;
         link.setAttribute(
           "download",
-          `${data.mode}_${data.min_wavenumber_range}_${data.max_wavenumber_range}.spec`
+          `${data.database}_${molecules}_${data.min_wavenumber_range}_${data.max_wavenumber_range}cm-1_${data.tgas}K_${data.pressure}atm.spec`
```

Comment:
What happens if there are nonequilibrium conditions ? 
Then you'll need Tvib, Trot

Reply:
also, what happens if there are multiple molecules ? It may because complicated to look over all molecules, just write "gas_mixture" if N molecules > 1 ?

Reply:
I think @erwanp it is great to put this way because that is going to show `undefined`

Reply:
Your implementation does not work if downloading a Spectrum of multiple molecules. I tried it